apps/web/src/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -89,6 +91,34 @@ const DashboardPage = () => {
     },
   ];
 
+  function getDuplicateInterns() {
+    const fullNames =
+      internsWithStatus?.map(
+        (i) =>
+          `${i.firstName.toLocaleLowerCase()} ${i.lastName.toLocaleLowerCase()}`,
+      ) || [];
+
+    const duplicateNames = fullNames.filter(
+      (name, index) => fullNames.indexOf(name) !== index,
+    );
+
+    const duplicateInterns: InternWithStatus[] = [];
+    internsWithStatus?.forEach((intern) => {
+      if (
+        duplicateNames.includes(
+          `${intern.firstName.toLocaleLowerCase()} ${intern.lastName.toLocaleLowerCase()}`,
+        )
+      ) {
+        duplicateInterns.push(intern);
+      }
+    });
+
+    return duplicateInterns;
```

Comment:
Možda je ovako jednostavnije:   
   ```
    const duplicateInterns = internsWithStatus?.filter(
      (i) =>
        internsWithStatus?.filter(
          (i2) => i2.firstName == i.firstName && i2.lastName == i.lastName,
        ).length > 1,
    );

    return duplicateInterns || [];
    ```

Reply:
i onda još tu staviš da gleda samo profile kojima je status pending, jer ako je neko duplikat mi ga izbacimo iz appa

Reply:
pa eto više onda nije duplikat
i kako ne bi morali otvarat to svaki put, možeš i stavit da piše negdje (prije otvaranja tog viewa), broj duplikata